Classes/Service/Tika/ServerService.php

```diff
@@ -59,17 +69,19 @@ class ServerService extends AbstractService
      */
     protected function initializeService()
```

Comment:
As we now added return type declarations for the other methords, we also could remove the `@return void` annotations and define the return type `void`. I think this would be consequent

Reply:
Return type added and the doc comments are now up to date.